bosh-stemcell/spec/os_image/opensuse_leap_spec.rb

```diff
@@ -0,0 +1,151 @@
+require 'spec_helper'
+
+describe 'openSUSE leap OS image', os_image: true do
+  it_behaves_like 'every OS image'
+  it_behaves_like 'a openSUSE based OS image'
+  it_behaves_like 'a systemd-based OS image'
+  it_behaves_like 'a Linux kernel 3.x based OS image'
+  it_behaves_like 'a Linux kernel module configured OS image'
+
+  context 'installed by base_opensuse' do
+    describe file('/etc/SuSE-release') do
+      it { should be_file }
+    end
+
+    describe file('/etc/locale.conf') do
+      it { should be_file }
+      its(:content) { should match 'en_US.UTF-8' }
+    end
+  end
+
+  context 'package signature verification (stig: V-38462)' do
+    describe command('grep nosignature /etc/rpmrc /usr/lib/rpm/rpmrc /usr/lib/rpm/redhat/rpmrc ~root/.rpmrc') do
+      its(:stdout) { should_not include('nosignature') }
+    end
+  end
+
+  context 'display the number of unsuccessful logon/access attempts since the last successful logon/access (stig: V-51875)' do
+    describe file('/etc/pam.d/common-password') do
+      its(:content){ should match /session required\tpam_lastlog\.so  showfailed/ }
+    end
+  end
+
+  context 'official Centos gpg key is installed (stig: V-38476)' do
+    describe command('rpm -qa gpg-pubkey* 2>/dev/null | xargs rpm -qi 2>/dev/null') do
+      its(:stdout) { should include('SuSE Package Signing Key') }
+    end
+  end
+
+  context 'gpgcheck must be enabled (stig: V-38483)' do
+    describe file('/etc/zypp/zypp.conf') do
```

Comment:
would prefer to have this be a positive test (check that gpgcheck, etc are set to a positive boolean value)

Reply:
This was done in 2878d9151690a90052683c9c7fba78e41098acaf